library/flink-library/src/test/java/uk/gov/gchq/gaffer/flink/operation/FlinkTest.java

```diff
@@ -122,7 +122,7 @@ public static Store createStore() {
 
     public static void verifyElements(final Graph graph) throws OperationException, InterruptedException {
         // Wait for the elements to be ingested.
-        Thread.sleep(2000);
+        Thread.sleep(3000);
```

Comment:
Is this failing at the moment?

Reply:
I think it originally failed once on the pipeline and intermittently on my personal machine (maybe it is slightly slower in some test runs) so the assertion runs too early and fails the test or breaks pipeline

I can change it back to 2000?

Reply:
No it's fine just might be worth noting if it's unreliable.